appveyor/run_test.ps1

```diff
@@ -12,6 +12,11 @@ python -c "import platform; print(platform.architecture())"
 
 $wc = New-Object 'System.Net.WebClient'
 nosetests -s -v --with-ignore-docstrings --with-xunit --xunit-file=unit_results.xml .\tests\unit
+
+$env:MONKEY_PATCH_LOOP=1
```

Comment:
A fast solution to this could be to unset MONKEY_PATCH_LOOP after those two command are run